lib/molecule_podman/playbooks/create.yml

```diff
@@ -101,26 +101,51 @@
       loop_control:
         label: "{{ item.name }}"
 
+    - name: Discover local podman networks
+      containers.podman.podman_network_info:
```

Comment:
This module is not available in Ansible 2.9, which means that people that want to use this driver will need to either use Ansible >= 2.10 or manually install the `containers.podman` collection.